__init__.py

```diff
@@ -58,7 +58,6 @@ def register():
         dt_op = getattr(dt_op, name)
     if hasattr(bpy.types, dt_op.idname()): # objects under bpy.ops are created on the fly, have to check that it actually exists a little differently
         raise RuntimeError("Another instance of Dream Textures is already running.")
-    
 
     bpy.types.Scene.dream_textures_requirements_path = EnumProperty(name="Platform", items=requirements_path_items, description="Specifies which set of dependencies to install", default='stable_diffusion/requirements-mac-MPS-CPU.txt' if sys.platform == 'darwin' else 'requirements-win-torch-1-11-0.txt')
```

Comment:
`TypeError: EnumProperty(..., default='requirements-win-torch-1-11-0.txt'): not found in enum members`
```suggestion
    bpy.types.Scene.dream_textures_requirements_path = EnumProperty(name="Platform", items=requirements_path_items, description="Specifies which set of dependencies to install", default='requirements-mac-MPS-CPU.txt' if sys.platform == 'darwin' else 'requirements-lin-win-colab-CUDA.txt')
```